### DISCUSSION TITLE
Spacing on Power and SID/CPU headers

### DISCUSSION THREAD

**Discussion by vk5ln** [frntc/Sidekick64#39] [Ideas]
Hello.

I'm wondering if the spacing on the Sidekick64 power input pins and SID/CPU pads could be changed to 2.54mm spacing in a future PCB revision?

Currently you can't use a single two pole connector for the SID/CPU wires, you need to use two single pole connectors.

It would also be very convenient to be able to use a single four pole connector for external power, the blank position could be blocked off to prevent it from being connected in reverse polarity.

Michael.

**Comment by frntc** [comment]
Hi,

the idea of having the SID/CPU pads with a different spacing was to make it impossible to accidentally put a jumper on the pins. But you're not the first one suggesting to use 2.54mm. This has been modified on the (smaller) Sidekick64 Zero PCB, but not yet on the larger one.

You mean the 5V pins on the left side of the PCB? These are NOT meant for providing 5V to the Sidekick. If this jumper is closed, then Sidekick is powered from the C64 (which has 5V on the expansion port). This is a different power circuit than the USB-power.

**Reply by vk5ln** [reply]
All noted on the 5V pins as not to be used for powering the Sidekick and the reasoning for not using a 2.54mm spacing on the SID/CPU pins.

At this stage I am only using the SID wire with my Sidekick but was thinking forward to when I start experimenting with the Kernal replacement function. It would convenient to have a single connector when removing/inserting the Sidekick64.

**Reply by frntc** [reply]
Next time when I touch the PCB, I'll make these changes! (also the other pin headers on the small PCB are aligned so that Sidekick-cases need less openings). As an ugly work around: I guess you could bend the two pins (or solder them slanted) to make the spacing 2.54mm ;-)

**Reply by banman11** [reply]
Maybe you can use something like a cutoff USB cable. That way you can have a very nice connection that can't be accidentally reversed. The added benefit would be you only need a round hole drilled into the case for the cable to protrude.
